server/utils/string.ts

```diff
@@ -6,3 +6,14 @@ export function hashString(input: string) {
   }
   return hash;
 }
+
+export function stringToHash(input: string) {
```

Comment:
I feel like this is confusingly named with the other function.

Reply:
Should we only keep the new one? I was not sure if changing the old one would break something.

Reply:
Or do you have a name suggestion?

Reply:
You implemented the new one because the old one was prone to collisions right?

I would probably just use a built in nodejs hash function if you need it on the server. The old simple hash function is shared by client and server.